elyra/templates/kfp_template.jinja2

```diff
@@ -22,6 +22,9 @@ def create_pipeline():
                                                       cos_dependencies_archive='{{ operation.cos_dependencies_archive }}',
                                                       pipeline_inputs={{ operation.pipeline_inputs }},
                                                       pipeline_outputs={{ operation.pipeline_outputs }},
+                                                      cpu_request='{{ operation.cpu_request }}',
+                                                      mem_request='{{ operation.mem_request }}G',
```

Comment:
Is there supposed to be a G at the end of this line?

Reply:
its a little weird I admit, looking for hands if we want to change it. 
We could add another parameter to the template rendering function in the kfp processor to pass the value with `G` appended to it, in this case it was just quicker to add a G to the template.